action.yml

```diff
@@ -0,0 +1,49 @@
+name: Sync Salmon Directory
+description: Syncs a directory to a Google Cloud bucket using rclone.
+author: 'deephaven'
+inputs:
+  source:
+    required: true
+    type: string
+    description: 'The source directory to sync.'
+  destination:
+    required: true
+    type: string
+    description: 'The destination directory to sync. Relative to the bucket.'
+  project_number:
+    required: true
+    type: string
+    description: 'The Google Cloud project number.'
+  bucket:
+    required: true
+    type: string
+    description: 'The Google Cloud bucket to sync to.'
+  credentials:
+    required: true
+    type: string
+    description: 'The Google Cloud credentials.'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup rclone
+      uses: AnimMouse/setup-rclone@v1
+      with:
+        version: v1.68.1
+
+    - name: Decode credentials
+      shell: bash
+      run: |
+        echo $RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS_ENCODED | base64 -d > $HOME/credentials.json
```

Comment:
Minor, but I prefer using full flags (`--decode`) in CI/scripts so it's more clear what the flag does